hat/assets/js/apps/Iaso/domains/projects/components/ProjectFeatureFlags.tsx

```diff
@@ -29,14 +31,43 @@ const ProjectFeatureFlags: FunctionComponent<Props> = ({
     featureFlags,
     isFetchingFeatureFlag,
 }) => {
+    const { formatMessage } = useSafeIntl();
     const options = React.useMemo(
         () =>
-            featureFlags?.map(fF => ({
-                label: fF.name,
-                value: fF.id,
-            })),
-        [featureFlags],
+            featureFlags?.map(fF => {
+                return {
+                    label: fF.name,
+                    value: fF.id,
+                    tooltip:
+                        formatMessage(
+                            MESSAGES[`${fF.code.toLowerCase()}_tooltip`],
+                        ) || fF.name,
+                };
+            }),
+        [featureFlags, formatMessage],
+    );
+    const renderOption = (props, option) => (
+        <Tooltip key={option.value} title={option.tooltip} disableInteractive>
+            <span {...props}>{option.label}</span>
+        </Tooltip>
     );
+
+    const renderTags = (value, getTagProps) =>
+        value.map((option, index) => (
+            <Tooltip
+                // eslint-disable-next-line react/no-array-index-key
+                key={index}
```

Comment:
Is there nothing in `value` that could be used as unique `key`. It works with `index` but it's good practice to avoid using it when possible.

Reply:
Thank you